source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -61,65 +63,77 @@ content: |
      * - Parameter
        - Description
 
-     * - Description
+     * - ``Description``
```

Comment:
Question for tech review: Does case matter? Should we stick to one style? We use first letter capitalized here and all caps below.

Reply:
I've tested and found the params to be case-insensitive on all platforms. Still, I think it would look better if we stick to all caps for all the params.